src/main/java/com/fasterxml/jackson/databind/cfg/JsonNodeFeature.java

```diff
@@ -61,8 +61,18 @@ public enum JsonNodeFeature implements DatatypeFeature
      *
      * @since 2.15
      */
-    STRIP_TRAILING_BIGDECIMAL_ZEROES(true)
-    ;
+    STRIP_TRAILING_BIGDECIMAL_ZEROES(true),
+
+    /**
+     *
+     * Feature that determines whether to fail on attempt to coercing {@code NaN} into {@link java.math.BigDecimal}
+     * because {@link com.fasterxml.jackson.databind.DeserializationFeature#USE_BIG_DECIMAL_FOR_FLOATS} is enabled.
```

Comment:
Should elaborate bit more: basically, that choices are Fail or Do-not-convert.

Other than that I think this setting makes sense.

Reply:
Revised, hopefully explains more now :))